src/partition.rs

```diff
@@ -102,6 +102,7 @@ impl BlockSize {
 
 /// Transform Size
 #[derive(Copy, Clone, Debug, PartialEq, PartialOrd)]
+#[repr(C)]
```

Comment:
Why did this become necessary?

Reply:
It's not, I can drop it.

Reply:
Er, actually, it's because the fht functions take it now. I think it's not technically necessary as it works without it, but I want to be sure.